src/components/UserInfo.js

```diff
@@ -9,15 +9,14 @@ export default class UserInfo {
       profileName: this._name.textContent,
       profileJob: this._job.textContent,
     };
-    console.log(profileData);
+
     return profileData;
   }
 
```

Comment:
Needs correction: it also needed to implement the method for updating the user avatar

Reply:
Done